### ISSUE TITLE
Automatic Network Switch

### ISSUE THREAD

**Issue by breezeight** [techstationspace/kuoricinidao#25]
# Context

If you forget to switch network the application only reads you wallet, you are connect to Metamask:
* you see your wallet id 
* you don't see your profile name

Step to reproduce the above behaviour: connect with eth network, go to https://pilot.kuori.app/profile.html .

A great improvement would be implementing the automatic network switch:
<img width="362" alt="Screenshot 2022-09-04 at 15 29 45" src="https://user-images.githubusercontent.com/17023/188316352-7129bf5a-e769-4163-918f-034e7cdee4e7.png">

# Implementation

This blog post describe how to implement this feature:
https://davidkathoh.medium.com/programatically-switch-network-on-metamask-e9a44525cab

**Comment by fabianchatwin** [comment]
is this related to issue #27 ?

**Comment by goldmark00000** [comment]
No, the issue #27  is about managing your wallet connection errors. This issue says that if the wallet is on a network other than polygon, it will be changed automatically.
